### ISSUE TITLE
Display read-only and write-only labels when using SchemaDefinition

### ISSUE THREAD

**Issue by cmeeren** [Redocly/redoc#991]
It's great to have the new SchemaDefinition! Thanks for all your hard work @nanov  and @RomanHotsiy! 🙏

It would be fantastic to have it display "read-only" and "write-only" labels for relevant properties.

**Comment by cmeeren** [comment]
I'm no JS dev, but might this be so simple to implement that I could make a PR with some light guidance?

**Comment by adamaltman** [comment]
Are you thinking it would display here:

![image](https://user-images.githubusercontent.com/1161871/66714575-cb5bd980-ed7d-11e9-841b-7d870629bbab.png)

**Comment by cmeeren** [comment]
Yes, exactly. That would be perfect IMHO. Similar to the current `required`.

![image](https://user-images.githubusercontent.com/7766733/66714972-f534f080-edbd-11e9-824c-3774eb2e38b2.png)

**Comment by adamaltman** [comment]
And if it is required and write-only, for example?

**Comment by cmeeren** [comment]
Display both `required` and `write-only`?

**Comment by adamaltman** [comment]
With a line break? or space? or an ` and ` as the separator?

**Comment by cmeeren** [comment]
I think I'd prefer line break or space rather than `and`, but I don't care that much.

**Comment by cmeeren** [comment]
Any progress on this? Anything I can do to help? (As previously said, I'm no JS dev, but might this be so simple to implement that I could make a PR with some light guidance?)

**Comment by cmeeren** [comment]
Hi! Hope everyone's fine. It's been a while since the last update here, which seemed promising. Any chance to get this through the door? 😄

**Comment by RomanHotsiy** [comment]
Hey @cmeeren, unfortunately, no progress on this yet.

But thanks for the reminder. I will prfioritize this as I find it useful and easy-to-add.

**Comment by cmeeren** [comment]
Very happy to hear that! As previously said, I'm not really a JS dev, but let me know if I can help. The lack of this feature is itching me something awful at the moment 😅

**Comment by potatolianyi** [comment]
Hi Redoc team, is there any update on this?
